src/solvers/prop/prop_conv.h

```diff
@@ -22,6 +22,9 @@ class prop_convt:public decision_proceduret
 
   using decision_proceduret::operator();
 
+  // handle
```

Comment:
Maybe the notes from the roadmap (explaining that this could be a `literal_exprt` or a symbol or a BDD node or ...) can be copied in here instead of this comment?

Reply:
Shouldn't that be abstract here and implemented in `prop_conv_solvert`? Only `prop_conv_solvert` would use `literal_exprt` other implementations of `prop_convt` might not.